test/e2e_node/eviction_test.go

```diff
@@ -199,7 +199,9 @@ var _ = SIGDescribe("LocalStorageEviction", framework.WithSlow(), framework.With
 		runEvictionTest(f, pressureTimeout, expectedNodeCondition, expectedStarvedResource, logDiskMetrics, []podEvictSpec{
 			{
 				evictionPriority: 1,
-				pod:              diskConsumingPod("container-disk-hog", lotsOfDisk, nil, v1.ResourceRequirements{}),
+				// TODO(#127864): Due to a race condition between eviction manager and containerd GC,
```

Comment:
Nit: Container runtime may not immediate free up the resources after the pod eviction, causing the test to fail. We provision an emptyDir volume to avoid relying on the runtime behavior.

Reply:
Done.